docs/sce_file_contents.md

```diff
@@ -136,8 +136,11 @@ metadata(sce) # experiment metadata
 | `singler_results` | If cell typing with `SingleR` was performed, the full result object returned by `SingleR` annotation. Only present for `processed` objects |
 | `singler_reference` | If cell typing with `SingleR` was performed, the name of the [`celldex`](http://bioconductor.org/packages/release/data/experiment/html/celldex.html) reference dataset used for annotation. Only present for `processed` objects |
 | `singler_reference_label` | If cell typing with `SingleR` was performed, the name of the label in the reference dataset used for annotation. Only present for `processed` objects |
+| `singler_reference_version`  | If cell typing with `SingleR` was performed with ontology labels, the [`celldex`](http://bioconductor.org/packages/release/data/experiment/html/celldex.html) package version used for the reference dataset |
```

Comment:
Just to note that I'm going to file a PR today with adding this into the SCE object. I am adding both package and version so it will be `celldex_1-12-1` or whatever the version is. So we might update this once we make a decision regarding how we are storing this information.

Reply:
Ok, I'll circle back once that PR is set then (we can block this PR as needed, probably dont _need_ to for this situation)

Reply:
And we're back! In the end, we did the following. Names here still need to be updated, but the components are not changed.

https://github.com/AlexsLemonade/scpca-nf/blob/b4e8cc860ea7a40f730fd6807e12f0c89f87dfb1/bin/add_celltypes_to_sce.R#L140-L144

https://github.com/AlexsLemonade/scpca-nf/blob/b4e8cc860ea7a40f730fd6807e12f0c89f87dfb1/bin/add_celltypes_to_sce.R#L189-L193